data_runner/manifest.json

```diff
@@ -0,0 +1,36 @@
+{
+  "manifest_version": "2.0.0",
+  "app_uuid": "ad7b5a3c-497d-45e0-9bcf-50f2d1365247",
+  "app_id": "data-runner",
+  "classifier_tags": [
+    "Supported OS::Linux",
+    "Supported OS::Mac OS",
+    "Supported OS::Windows",
+    "Category::Games",
```

Comment:
```suggestion
```

We don't have this as a supported category.

Reply:
👍 what's the process for new categories? We might want some for these rather unusual things getting added